router/batchrouter/asyncdestinationmanager/bing-ads/offline-conversions/bulk_uploader.go

```diff
@@ -66,6 +66,17 @@ func (b *BingAdsBulkUploader) Transform(job *jobsdb.JobT) (string, error) {
 			return payload, err
 		}
 	}
+
+	// Validate conversionTime format as ISO 8601 (RFC3339)
+	// Bing ads supports timestamp of the format 2020-01-01T00:00:00.10000Z
+	conversionTimeStr, ok := fields["conversionTime"].(string)
+	if !ok {
+		return payload, fmt.Errorf("conversionTime field is either not string or an empty string")
```

Comment:
this message is wrong, it won't fail when empty string , you need to add additional checks if you want to error for empty string

Reply:
The issue was that the test was expecting the error message "conversionTime field is not a string" from the type assertion check in lines 72-75 of `bulk_uploader.go`, but the actual error being returned was "conversionTime field is either not string or an empty string" (from the `validateField` function in `util.go`).